src/styles/main.scss

```diff
@@ -1,3 +1,5 @@
-body {
-  margin: 0;
-}
+@import "modules/page";
+@import "modules/header";
+@import "modules/cards";
+@import "modules/stars";
+@import "modules/content";
\ No newline at end of file
```

Comment:
There is no newline at the end of file. It is a common convention to put a new line at the end of file. It's a standard defined in POSIX.